tests/test_pois.py

```diff
@@ -0,0 +1,227 @@
+        self.assertEqual(sorted(response_story_ids), sorted(poi_create_complete['story_ids']))
+
+    def test1_3_get_pois_by_map_year(self):
+        map_year = poi_create_complete['map_year']
+        r = requests.get('http://127.0.0.1:5000/pois?map_year={}'.format(map_year))
+        response = r.json()
+        self.assertEqual(response['code'], 200)
+        for i in response['result']['pois']:
+            self.assertEqual(i['map_year'], map_year)
+
+    def test1_4_get_pois_by_story_id(self):
+        story_ids = poi_create_complete['story_ids']
+        for curr_story_id in story_ids:
+            r = requests.get('http://127.0.0.1:5000/pois?story_id={}'.format(curr_story_id))
+            response = r.json()
+            self.assertEqual(response['code'], 200)
+            # created_object_included is used to determine if the POI we previously created is included
+            # in the list of POIs returned after we get request based on story_ids
+            created_object_included = False
+            for curr_poi in response['result']['pois']:
+                if curr_poi['name'] == poi_create_complete['name']:
+                    created_object_included = True
+            self.assertTrue(created_object_included)
+
+    def test1_5_update_poi(self):
+        poi_id = int(os.environ.get('POI_ID'))
+        r = requests.put('http://127.0.0.1:5000/pois/{}'.format(poi_id), json=poi_update_complete)
+        response = r.json()
+        self.assertEqual(response['code'], 200)
+        poi = response['result']['poi']
+        for key in poi_create_complete:
+            # Date in response is a date object, while date in json is just a string
+            # self.assertEqual(response['result']['poi']['date'], poi_create_complete['date'])
+            if key not in ['date', 'links', 'media', 'story_ids']:
+                self.assertEqual(poi[key], poi_create_complete[key])
+            elif key == 'links':
+                for j in range(len(poi_update_complete[key])):
+                    self.assertEqual(poi[key][j]['link_url'], poi_update_complete[key][j]['link_url'])
+                    self.assertEqual(poi[key][j]['display_name'], poi_update_complete[key][j]['display_name'])
+            elif key == 'media':
+                for j in range(len(poi_update_complete[key])):
+                    self.assertEqual(poi[key][j]['content_url'], poi_update_complete[key][j]['content_url'])
+                    self.assertEqual(poi[key][j]['caption'], poi_update_complete[key][j]['caption'])
+        response_story_ids = [i['_id'] for i in response['result']['poi']['stories']]
+        self.assertEqual(sorted(response_story_ids), sorted(poi_update_complete['story_ids']))
+
+    def test1_6_update_none(self):
+        poi_id = int(os.environ.get('POI_ID'))
+        r = requests.put('http://127.0.0.1:5000/pois/{}'.format(poi_id), json=poi_update_none)
+        response = r.json()
+        # Successfully updates nothing
```

Comment:
it would be good to check the attributes of the response and actually make sure nothing has changed. technically, if the endpoint logic changes a bunch of stuff but still returns a `status` of `200`, then this test will pass